src/com/lftechnology/java/training/sanish/palindrome/Palindrome.java

```diff
@@ -0,0 +1,57 @@
+package com.lftechnology.java.training.sanish.palindrome;
+
+import java.util.logging.Logger;
+
+/**
+ * Provide palindrome calculation functions
+ * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+ */
+public class Palindrome {
+	/**
+	 * Calculate and return largest palindrome number made from the product of two provided digit numbers
+	 * 
+	 * @param digitLength digit length of two product from which have find largest palindrome number
+	 * @return largest palindrome number
+	 * @throws Exception Digit length greater than 4 not supported
+	 * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+	 */
+	public static int getLargestPalindrome(int digitLength) throws Exception {
+		final Logger LOGGER = Logger.getLogger(Main.class.getName());
+		int maxNum = (int) Math.pow(10, digitLength) - 1;
+		int minNum = (int) Math.pow(10, digitLength - 1);
+		if (digitLength <= 4) {
+			for (int i = maxNum; i > minNum; i--) {
+				for (int j = maxNum; j > minNum; j--) {
+					int multipleNum = j * i;
+					if (isPalindromNum(multipleNum)) {
```

Comment:
mutipleNum should be intialized outside the loop. Cause the value will be initialized every time the loop run.